bin/mixer_codegen.sh

```diff
@@ -98,6 +100,8 @@ if [ "$opttemplate" = true ]; then
     "google/protobuf/any.proto:github.com/gogo/protobuf/types"
     "gogoproto/gogo.proto:github.com/gogo/protobuf/gogoproto"
     "google/protobuf/duration.proto:github.com/gogo/protobuf/types"
+    "google/protobuf/timestamp.proto=github.com/gogo/protobuf/types"
+    "google/rpc/status.proto=github.com/gogo/googleapis/google/rpc"
```

Comment:
Did you mean to add `struct.proto` here too, or is that not needed in templates?

Reply:
done.